pixi.toml

```diff
@@ -11,3 +11,4 @@ version = "0.6.1"
 [dependencies]
 python = "3.12.*"
 anaconda-client = "==1.12.3"
+jq = "==1.7.1"
```

Comment:
Please do not add additional dependencies to the default environment. This is the `pixi` manifest file for the upload action and additional dependencies should not be added to it.

I would recommend making a separate `pixi` project in a subdirectory.

Regardless, no dependency should be added without also updating a corresponding lock file.

Reply:
Thanks for this – I added a separate manifest at `remove-wheels/pixi.toml` like you requested in 4a3497d3b81ce7eee22d4b632fd3d6471377b9de and added a lockfile for it in 71ede798fffb686daf5bb5d49096af579859ac06.

Reply:
> I'm assuming that the lockfile is cross-platform

@agriyakhetarpal Yes, that is what the 

```toml
platforms = ["linux-64", "osx-64", "osx-arm64"]
```

that you added does for you. :+1: 


The `pixi` docs are very useful and approachable: https://pixi.sh/ So I'd recommend a speed read via searching it for key words.

Reply:
Awesome! Yes, I was skimming the docs in the meantime – they're neat :)